### PR TITLE
control: add support for setting the direction of the MOVE effect

### PR DESCRIPTION
**Pull request by Djelibeybi**
Includes an update to the docs for both the task and Interactor
command to provide a little more clarity on how to set this and
other options.

Also updated the test for this effect.

Signed-off-by: Avi Miller <me@dje.li>

### CODE REVIEW

**Comment by delfick** [comment]
Nice :)

Though assignment operator is 3.8+ and photons in 3.7+ till June next year
(when python stops making security updates for it).

But whether I be lazy and just make it 3.9+ wouldn't be hard to convince me
of hahaha

**Comment by Djelibeybi** [comment]
> But whether I be lazy and just make it 3.9+ wouldn't be hard to convince me of hahaha

Home Assistant is already 3.10+ which explains why I used the operator, I guess. But this is not significant enough a change to warrant a dependency bump, so I fixed it. :)

**Comment by Djelibeybi** [comment]
Oh, I love the fact that GitHub now lets you commit a suggestion by clicking a button. That's very handy.

File: modules/tests/photons_control_tests/test_multizone.py [hunk 1]
```diff
@@ -780,7 +780,9 @@ def compare_received(self, by_light):
             )
 
         async it "has options", sender:
-            msg = SetZonesEffect("move", speed=5, duration=10, power_on_duration=20)
+            msg = SetZonesEffect(
+                "move", speed=5, duration=10, direction="RIGHT", power_on_duration=20
```
Comment (delfick) [inline modules/tests/photons_control_tests/test_multizone.py:784]:
```suggestion
                "move", speed=5, duration=10, direction="LEFT", power_on_duration=20
```

and then change the test to show it made a difference

**Comment by Djelibeybi** [comment]
GitHub's auto-merging is actually bad. Apologies. I'll pull and make the changes properly.

**Comment by delfick** [comment]
> GitHub's auto-merging is actually bad. Apologies. I'll pull and make the changes properly.

hhahahah

File: modules/photons_control/multizone.py [hunk 2]
```diff
@@ -264,6 +265,18 @@ def SetZonesEffect(effect, power_on=True, power_on_duration=1, reference=None, *
 
     options["type"] = typ
     options["res_required"] = False
+
+    direction = options.pop("direction", None)
+    if isinstance(direction, str):
+        direction = Direction.__members__.get(direction.upper())
+    if isinstance(direction, Direction):
+        options["parameters"] = {"speed_direction": direction}
+    if direction is not None:
```
[the remark this answers is not in the document]
Comment (Djelibeybi) [reply modules/photons_control/multizone.py:274]: BTW, you can make them span multiple lines with Shift-Click+Drag. It's very nifty.

[the remark this answers is not in the document]
Comment (delfick) [reply modules/photons_control/multizone.py:274]: fancy

[the remark this answers is not in the document]
Comment (delfick) [reply modules/photons_control/multizone.py:274]: actually, you drag the blue comment button and then you can make a comment with a multiline suggestion

**Review by delfick — APPROVED** [review]

**Comment by delfick** [comment]
https://github.com/delfick/photons/runs/7034985825?check_suite_focus=true

**Pull request merged**